app/components/EventItem/index.tsx

```diff
@@ -1,30 +1,67 @@
-import moment from 'moment-timezone';
 import { Link } from 'react-router-dom';
+import Icon from 'app/components/Icon';
 import { Image } from 'app/components/Image';
 import { Flex } from 'app/components/Layout';
 import Pill from 'app/components/Pill';
 import Tag from 'app/components/Tags/Tag';
 import Time from 'app/components/Time';
+import Tooltip from 'app/components/Tooltip';
 import type { Event, EventTime } from 'app/models';
 import { colorForEvent } from 'app/routes/events/utils';
-import { eventStatus, eventAttendance } from 'app/utils/eventStatus';
+import { eventAttendanceAbsolute } from 'app/utils/eventStatus';
 import styles from './styles.css';
 import type { ReactNode } from 'react';
 
-type AttendanceProps = {
-  event: Event;
+export type EventStyle = 'default' | 'extra-compact' | 'compact';
+
+type statusIconProps = {
+  status: string;
+  icon: string;
+  color: string;
+  tooltip: string;
+};
+
+const eventStatusObject = (event: Event): statusIconProps => {
+  const { isAdmitted, eventStatusType } = event;
+
+  switch (eventStatusType) {
+    case 'NORMAL':
+    case 'INFINITE':
+      if (isAdmitted) {
+        return {
+          status: 'Admitted',
+          icon: 'checkmark-circle-outline',
+          color: '#1ec91e',
+          tooltip: 'Du er påmeldt',
+        } as statusIconProps;
+      }
+      return {
+        status: 'Waitlist',
+        icon: 'timer-outline',
+        color: '#fc9003',
+        tooltip: 'Du er på ventelisten',
+      } as statusIconProps;
+    default:
+      return {
+        status: 'Error',
+        icon: 'help-outline',
+        color: 'blue',
+        tooltip: 'Det har oppstått en feil',
+      } as statusIconProps;
+  }
 };
-export type EventStyle = 'default' | 'extra-compact';
 
-const Attendance = ({ event }: AttendanceProps) => {
-  const attendance = eventAttendance(event);
+const Attendance = ({ event }) => {
+  const attendance = eventAttendanceAbsolute(event);
   return (
-    attendance && (
+    !!attendance && (
       <Pill
         style={{
           marginLeft: '5px',
+          marginBottom: '5px',
           color: 'black',
           whiteSpace: 'nowrap',
+          alignItems: 'center',
```

Comment:
Is the content not aligned already?

Reply:
I couldn't find a way of removing the inline styling here, as it would force the class to be overwritten and therefore lose a lot of the attributes.
Is there a way around this issue?